.travis.yml

```diff
@@ -25,11 +25,8 @@ before_install:
             sudo docker pull welder/web-cockpit:latest
         fi
         if [ "$COMMAND" == "end-to-end-test" ]; then
-            sudo docker pull welder/web-nodejs:latest
             sudo docker pull welder/web:latest
             sudo docker pull welder/web-e2e-tests:latest
-        else
-            npm install
```

Comment:
IMO this should not be deleted. From what I can tell Travis does a `npm install` before it executed the test targets (e.g. eslint) simply because the build is labeled as Nodejs one. 

This is an artifact of the build/test infrastructure which we should not rely upon. 

Either revert or explain why this line needs to be deleted.

Reply:
npm install is the default command in Travis install stage. So we do not put a duplicated one here.
You can find "npm install" from Travis log of this PR.

Reply:
Yes, I know this and I don't want to rely on it implicitly. Every command should be clearly visible in our own source code, not hidden into log files or 3rd party documentation.